src/wp-includes/post.php

```diff
@@ -6480,6 +6480,25 @@ function wp_delete_attachment_files( $post_id, $meta, $backup_sizes, $file ) {
 					$deleted = false;
 				}
 			}
+			// Check for alternate size mime types in the sizeinfo['sources'] array to delete.
+			if ( isset( $sizeinfo['sources'] ) && is_array( $sizeinfo['sources'] ) && 1 < sizeof( $sizeinfo['sources'] ) ) {
```

Comment:
Why not use `count()` instead of `sizeof()`? That's a bit odd. Also, we don't need to use Yoda conditions on greater/lower comparisons, it's just useful for equal checks.

Reply:
updated